content/cli/orch_cli/apply.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: bt_wiki
+title: apply
+category: Docs
+draft: false
+abstract: Cloudify's Command-Line Interface
+aliases: /cli/apply/
+---
+
+The `cfy apply` command is used to install/update a deployment using {{< param product_name >}} manager without having to manually go through the process of uploading a blueprint, creating a deployment, and executing a workflow.
+`cfy apply` command using `cfy install` or `cfy deployments update` logic depends on the existence of `DEPLOYMENT_ID`. 
+
+It is recommended to read about [`cfy install`]({{< relref "cli/orch_cli/install.md" >}}) and [`cfy deployments update`]({{< relref "cli/orch_cli/deployments.md#update" >}}) in order to understand `cfy apply` command.
+
+`cfy apply` logic:
+
+1. Check for `BLUPRINT_PATH` and `DEPLOYMENT_ID`.
+
+2. If `BLUPRINT_PATH` is missing use default value and infer `DEPLOYMENT_ID`(explained in usage section).
```

Comment:
```suggestion
2. If `BLUPRINT_PATH` is missing, use the default value and infer `DEPLOYMENT_ID`(explained in the usage section).
```
How about linking to the usage section?

Reply:
It's two rows below I don't see the point for that.

Reply:
OK, what about the suggestion?

Reply:
Done.